aws/adapter.go

```diff
@@ -552,21 +552,21 @@ func (a *Adapter) UpdateTargetGroupsAndAutoScalingGroups(stacks []*Stack) {
 	for _, asg := range nonTargetedASGs {
 		// This call is idempotent and safe to execute every time
 		if err := updateTargetGroupsForAutoScalingGroup(a.autoscaling, a.elbv2, nil, asg.name, ownerTags); err != nil {
-			log.Errorf("UpdateTargetGroupsAndAutoScalingGroups() failed to attach target groups to ASG '%s': %v", asg.name, err)
+			log.Errorf("failed to update target groups for non-targeted autoscaling group %q: %v", asg.name, err)
```

Comment:
Log messages should start with upper case

Reply:
Fixed